resources/shellHelperFunctions.sh

```diff
@@ -257,7 +257,10 @@ set_npm_version () {
   # release of Keyman Developer -- these two versions should be in sync. Because this
   # is a large repo with multiple projects and build systems, it's better for us that
   # individual build systems don't take too much ownership of git tagging. :)
-  npm --no-git-tag-version --allow-same-version version "$version" || fail "Could not set package version to $version."
+  pushd . > /dev/null
+  cd "${KEYMAN_ROOT}"
+  npm run version "$version"  || fail "Could not set package versions to $version."
+  popd > /dev/null
```

Comment:
Compare against the package script being called:

https://github.com/keymanapp/keyman/blob/869899fbacb425adf1e3f24411dfdbcfbb6a0504/package.json#L8-L11

The one component that got dropped:  the `--allow-same-version` flag.  Lerna doesn't support it and will throw an error when it's present.

Fortunately, that's because it's simply not _optional_ to Lerna; it's always active.  I say this based on the following test:

<img width="697" alt="Screen Shot 2021-01-19 at 11 57 44 AM" src="https://user-images.githubusercontent.com/25213402/104990144-9c3b1a00-5a4e-11eb-874b-bc1f663f88ff.png">

<img width="697" alt="Screen Shot 2021-01-19 at 12 06 40 PM" src="https://user-images.githubusercontent.com/25213402/104990244-d0163f80-5a4e-11eb-975d-7f762b42ec9c.png">

No errors resulted from this call; `lerna` is perfectly happy if the current version is specified as the new version.

Reply:
> The one component that got dropped: the `--allow-same-version` flag

I see these flags got added:
`--amend --yes --force-publish`

Just checking `--force-publish` doesn't have the potential to bite us...

Reply:
That's fair - I definitely had to double-check at first.

https://github.com/lerna/lerna/tree/main/commands/version#--force-publish

Reply:
> This will skip the lerna changed check for changed packages and forces a package that didn't have a git diff change to be updated.